doc/api/http2.md

```diff
@@ -3114,7 +3114,7 @@ changes:
     description: This method now returns a reference to `ServerResponse`.
 -->
 
-* `data` {string|Buffer}
+* `data` {string|Buffer|Uint8Array}
```

Comment:
`Buffer` extends `Uint8Array`, right?
```suggestion
* `data` {string|Uint8Array}
```

Reply:
It does (from `FastBuffer` which extends `Uint8Array`), but I'd like to keep this as it is for now, since a) there are many instances in docs and js following the same convention and b) we can't always expect the user to know this so it's a better idea to mention that explicitly.